enterprise/advanced_tables.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Advanced Tables plugin
+title_nav: Advanced Tables
+description: Add sorting functionality to tables.
+keywords: sort tables advanced advtable
+---
+
+The initial release of Advanced Tables adds sorting functionality to tables in {{site.productname}}. Tables can be sorted by row or column values using:
```

Comment:
Should we have initial release mentioned here, as that feels like wording that should really only be in the release notes?

Reply:
Also it can now only be sorted by columns using the menus directly, so not sure if that could be confusing?

It's also possible to use the sort option in the table toolbar button.

Reply:
fixed